x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/transport/netty4/SimpleSecurityNetty4ServerTransportTests.java

```diff
@@ -525,7 +525,7 @@ public void testClientChannelUsesSeparateSslConfigurationForRemoteCluster() thro
 
         final Settings fcSettings = Settings.builder()
             .put("remote_cluster_server.enabled", "true")
-            .put("remote_cluster.port", "9999")
+            .put("remote_cluster.port", "0")
```

Comment:
Not sure why I had it as `9999` previously, everywhere else we have used `0` for random port ...